src/panels/lovelace/cards/hui-picture-glance-card.js

```diff
@@ -134,54 +136,54 @@ class HuiPictureGlanceCard extends NavigateMixin(LocalizeMixin(EventsMixin(Polym
       (config.state_image && !config.entity)) {
       throw new Error('Invalid card configuration');
     }
-
+    this._configEntities = processConfigEntities(config.entities);
     this._config = config;
   }
 
   _computeEntitiesDialog(hass, config, entitiesService) {
```

Comment:
(hass, configEntities, entitiesService)

Reply:
Not possible, `config.force_dialog` requires  config as well